lumigator/python/mzai/backend/backend/api/routes/workflows.py

```diff
@@ -0,0 +1,67 @@
+from uuid import UUID
+
+from fastapi import APIRouter, BackgroundTasks, status
+from lumigator_schemas.workflows import (
+    WorkflowCreate,
+    WorkflowDetailsResponse,
+    WorkflowResponse,
+    WorkflowSummaryResponse,
+)
+
+from backend.api.deps import WorkflowServiceDep
+
+router = APIRouter()
+
+
+@router.post("/", status_code=status.HTTP_201_CREATED)
+async def create_workflow(
+    service: WorkflowServiceDep, request: WorkflowCreate, background_tasks: BackgroundTasks
+) -> WorkflowResponse:
+    """A workflow is a single execution for an experiment.
+    A workflow can be a single job or a collection of jobs.
```

Comment:
Why not always a collection of jobs? A single job would be a list of 1 element.

Reply:
Ahhh great point! That is what I was thinking too, but my description is poor 😆 . I'll fix to clarify.